specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-10-20/examples/WorkbooksManagedList.json

```diff
@@ -0,0 +1,82 @@
+{
+  "parameters": {
+    "api-version": "2020-10-20",
+    "subscriptionId": "6b643656-33eb-422f-aee8-3ac119r124af",
+    "resourceGroupName": "my-resource-group",
+    "category": "workbook",
+    "sourceId": "/subscriptions/ad2f1a83-caac-4e21-9d2a-9ca3f87105e2/resourceGroups/Default-Web-WestUS/providers/Microsoft.Web/sites/MyTestApp-CodeLens1"
+  },
+  "responses": {
+    "200": {
+      "body": [
```

Comment:
list can have a mixture of resources with identities and without identities in the array